src/resources/views/list.blade.php

```diff
@@ -97,57 +91,77 @@
 @endsection
 
 @section('after_scripts')
-  	<!-- DATA TABLES SCRIPT -->
-    <script src="{{ asset('vendor/adminlte/plugins/datatables/jquery.dataTables.js') }}" type="text/javascript"></script>
-
-    <script src="{{ asset('vendor/backpack/crud/js/crud.js') }}"></script>
-    <script src="{{ asset('vendor/backpack/crud/js/form.js') }}"></script>
-    <script src="{{ asset('vendor/backpack/crud/js/list.js') }}"></script>
-
-    @if ($crud->exportButtons())
-    <script src="https://cdn.datatables.net/1.10.12/js/dataTables.bootstrap.min.js" type="text/javascript"></script>
-    <script src="https://cdn.datatables.net/buttons/1.2.2/js/dataTables.buttons.min.js" type="text/javascript"></script>
-    <script src="https://cdn.datatables.net/buttons/1.2.2/js/buttons.bootstrap.min.js" type="text/javascript"></script>
-    <script src="//cdnjs.cloudflare.com/ajax/libs/jszip/2.5.0/jszip.min.js" type="text/javascript"></script>
-    <script src="//cdn.rawgit.com/bpampuch/pdfmake/0.1.18/build/pdfmake.min.js" type="text/javascript"></script>
-    <script src="//cdn.rawgit.com/bpampuch/pdfmake/0.1.18/build/vfs_fonts.js" type="text/javascript"></script>
-    <script src="//cdn.datatables.net/buttons/1.2.2/js/buttons.html5.min.js" type="text/javascript"></script>
-    <script src="//cdn.datatables.net/buttons/1.2.2/js/buttons.print.min.js" type="text/javascript"></script>
-    <script src="//cdn.datatables.net/buttons/1.2.2/js/buttons.colVis.min.js" type="text/javascript"></script>
-    @endif
-
-    <script src="{{ asset('vendor/adminlte/plugins/datatables/dataTables.bootstrap.js') }}" type="text/javascript"></script>
-
-	<script type="text/javascript">
-	  jQuery(document).ready(function($) {
-
-      @if ($crud->exportButtons())
-      var dtButtons = function(buttons){
-          var extended = [];
-          for(var i = 0; i < buttons.length; i++){
-          var item = {
-              extend: buttons[i],
-              exportOptions: {
-              columns: [':visible']
-              }
-          };
-          switch(buttons[i]){
-              case 'pdfHtml5':
-              item.orientation = 'landscape';
-              break;
-          }
-          extended.push(item);
-          }
-          return extended;
-      }
-      @endif
-
-	  	var table = $("#crudTable").DataTable({
-        "pageLength": {{ $crud->getDefaultPageLength() }},
-        "lengthMenu": @json($crud->getPageLengthMenu()),
+	<!-- DATA TABLES SCRIPT -->
+  <script src="https://cdn.datatables.net/1.10.16/js/jquery.dataTables.min.js" type="text/javascript"></script>
```

Comment:
I don't like this indentation change. Makes this file much harder to review

Reply:
I agree. I don't see the same indentation in my files, though. Will try it again soon, but it looks like it might be just a Github GUI quirk.